pkg/preparer/setup.go

```diff
@@ -179,9 +179,8 @@ func getTLSConfig(certFile, keyFile, caFile string) (*tls.Config, error) {
 		certs = append(certs, cert)
 	}
 
-	var cas *x509.CertPool
+	cas := x509.NewCertPool()
```

Comment:
When the root CAs are unspecified, the system default CAs are used. That seems like preferable behavior to trusting nothing except what is explicitly configured.

Can you revert the change to these two lines? Removing the `if len(certs)...` block should be sufficient to prevent the exception.